fixWaas.js

```diff
@@ -0,0 +1,28 @@
+const { readFileSync, writeFileSync } = require("fs")
```

Comment:
Consider adding a short comment to this file which clarifies why do we need this 😉

Reply:
Good point! Added in https://github.com/guildxyz/guild.xyz/pull/1175/commits/2f1ec19b5e406a39c759186f27c9365a507bdbe5 :ayy: